protocol/bc/txtransaction.go

```diff
@@ -7,6 +7,29 @@ import (
 	"chain/errors"
 )
 
+type TxEntries struct {
```

Comment:
It would be great if, when the dust all settles, the main type we end up dealing with for a transaction still ends up being called `Tx`. If this is destined to be that type, can we find a way to name it that now (e.g. by putting these fields in the existing Tx type) or rename it later?